jenkins/Jenkinsfile-build

```diff
@@ -0,0 +1,11 @@
+library('private-pipeline-library')
+library('jenkins-shared')
+
+mavenSnapshotPipeline(usePublicSettingsXmlFile: true,
```

Comment:
Just another question of JDK6 vs JDK8.  
I'm assuming it's okay to move this package forward, but I just want to confirm.

Reply:
since the build worked, I figured that we're better off with Java 8 - I only want to use older java and/or maven if I absolutely have to... JDK6 is *supposed* to reach EOL at the end of '18 http://www.oracle.com/technetwork/java/javase/eol-135779.html

Reply:
Good point.  I'd prefer newer version too, just wanted to make sure.